technical_docs/Claims.md

```diff
@@ -150,7 +152,7 @@ To stageout to `/store/user/bbockelm`, a part of the CMS namespace at the CMS si
 }
 ```
 
-The implementation of `site` is purposely ambiguous; hence, the `aud` claim may be used to restrict a token to a specific endpoint:
+The implementation of `site` is purposely ambiguous; hence, the `aud` claim must be used to restrict a token to a specific endpoint:
```

Comment:
Can we just remove `site` for SciTokens 2.0?  I think we've just been using `aud` instead.